src/middlewared/middlewared/plugins/iscsi.py

```diff
@@ -800,7 +798,8 @@ async def clean_type_and_path(self, data, schema_name, verrors):
         if extent_type is None:
             return data
 
-        if extent_type == 'Disk':
+        # raw disks being shared via iscsi is only supported on freeBSD
+        if osc.IS_FREEBSD and extent_type == 'Disk':
```

Comment:
How would this happen even on FreeBSD ?

Reply:
Not sure I understand. This would happen if we release a core image based off master repo?

Reply:
You just removed all Disk from database so this would never be true and its easier/better to remove the code too?